app/javascript/pages/QuestionIndex.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import QuestionIndex from "./QuestionIndex";
+import mockQuestion from "../components/mockQuestion";
+import mockSession from "../components/mockSession";
+import Button from "@mui/material/Button";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("When QuestionIndex renders and a user is logged in", () => {
+  it(" renders a navlink and a div", () => {
+    const questionIndex = shallow(
+      <QuestionIndex
+        current_user={mockSession[0].current_user}
+        questions={mockQuestion}
+      />
+    );
+    expect(questionIndex.length).toEqual(1);
```

Comment:
I think you can get rid of this line. I'm pretty sure this would b e true even if nothing was rendered. So, it doesn't actually test anything.